### ISSUE TITLE
`envs` appears to be broken

### ISSUE THREAD

**Issue by owenRiddy** [typedclojure/typedclojure#121]
Hi again! It has been 12 months so I'm giving Typed Clojure another look (see https://github.com/typedclojure/typedclojure/issues/49). One of the problems I'm having is I've lost the `t/Long` type and it seems `t/envs` is broken again.

# project.clj

```
(defproject example "1.0.0-SNAPSHOT"
  :dependencies [[org.clojure/clojure "1.11.1"]
                 [org.typedclojure/typed.clj.checker "1.1.4"]
                 [org.typedclojure/typed.clj.runtime "1.1.4"]
                 ]
  :main ^:skip-aot example.core)
```

# src/example/core.clj

```
(ns example.core
  (:require
   [clojure.core.typed :as t]))

(defn -main
  [& _args]
  (prn "Starting Main!")
  (prn (t/envs)))
```

# lein run

```
$ lein run
"Starting Main!"
Registering annotations from typed.ann.clojure...
Execution error (ExceptionInfo) at clojure.core.typed.errors/int-error (errors.cljc:47).
Internal Error (typed/cljc/runtime/env_utils.clj:76:3) Cannot resolve type: s/Str
Hint: Is s/Str in scope in namespace `typed.cljc.runtime.env-utils`?
```

```
{:clojure.main/message
 "Execution error (ExceptionInfo) at clojure.core.typed.errors/int-error (errors.cljc:47).\nInternal Error (typed/cljc/runtime/env_utils.clj:76:3) Cannot resolve type: s/Str\nHint: Is s/Str in scope in namespace `typed.cljc.runtime.env-utils`?\n",
 :clojure.main/triage
 {:clojure.error/class clojure.lang.ExceptionInfo,
  :clojure.error/line 47,
  :clojure.error/cause
  "Internal Error (typed/cljc/runtime/env_utils.clj:76:3) Cannot resolve type: s/Str\nHint: Is s/Str in scope in namespace `typed.cljc.runtime.env-utils`?",
  :clojure.error/symbol clojure.core.typed.errors/int-error,
  :clojure.error/source "errors.cljc",
  :clojure.error/phase :execution},
 :clojure.main/trace
 {:via
  [{:type clojure.lang.Compiler$CompilerException,
    :message
    "Syntax error macroexpanding at (/tmp/form-init10603327060393245003.clj:1:74).",
    :data
    {:clojure.error/phase :execution,
     :clojure.error/line 1,
     :clojure.error/column 74,
     :clojure.error/source "/tmp/form-init10603327060393245003.clj"},
    :at [clojure.lang.Compiler load "Compiler.java" 7665]}
   {:type clojure.lang.ExceptionInfo,
    :message
    "Internal Error (typed/cljc/runtime/env_utils.clj:76:3) Cannot resolve type: s/Str\nHint: Is s/Str in scope in namespace `typed.cljc.runtime.env-utils`?",
    :data
    {:type-error :clojure.core.typed.errors/internal-error,
     :env
     {:line 76,
      :column 3,
      :file "typed/cljc/runtime/env_utils.clj",
      :ns typed.cljc.runtime.env-utils}},
    :at
    [clojure.core.typed.errors$int_error
     invokeStatic
     "errors.cljc"
     47]}],
  :trace
  [[clojure.core.typed.errors$int_error invokeStatic "errors.cljc" 47]
   [clojure.core.typed.errors$int_error invoke "errors.cljc" 42]
   [typed.clj.checker.parse_unparse$prs_error$f__12849
    invoke
    "parse_unparse.clj"
    131]
   [typed.clj.checker.parse_unparse$prs_error
    invokeStatic
    "parse_unparse.clj"
    135]
   [typed.clj.checker.parse_unparse$prs_error
    invoke
    "parse_unparse.clj"
    127]
   [typed.clj.checker.parse_unparse$prs_error
    invokeStatic
    "parse_unparse.clj"
    128]
   [typed.clj.checker.parse_unparse$prs_error
    invoke
    "parse_unparse.clj"
    127]
   [typed.clj.checker.parse_unparse$parse_type_symbol_default
    invokeStatic
    "parse_unparse.clj"
    1217]
   [typed.clj.checker.parse_unparse$parse_type_symbol_default
    invoke
    "parse_unparse.clj"
    1181]
   [typed.clj.checker.parse_unparse$eval13618$fn__13619
    invoke
    "parse_unparse.clj"
    1225]
   [clojure.lang.MultiFn invoke "MultiFn.java" 229]
   [typed.clj.checker.parse_unparse$parse_type
    invokeStatic
    "parse_unparse.clj"
    141]
   [typed.clj.checker.parse_unparse$parse_type
    invoke
    "parse_unparse.clj"
    137]
   [typed.clj.checker.parse_unparse$parse_clj$fn__12858
    invoke
    "parse_unparse.clj"
    158]
   [clojure.lang.AFn applyToHelper "AFn.java" 152]
   [clojure.lang.AFn applyTo "AFn.java" 144]
   [clojure.core$apply invokeStatic "core.clj" 667]
   [clojure.core$with_bindings_STAR_ invokeStatic "core.clj" 1990]
   [clojure.core$with_bindings_STAR_ doInvoke "core.clj" 1990]
   [clojure.lang.RestFn invoke "RestFn.java" 425]
   [typed.clj.checker.parse_unparse$parse_clj
    invokeStatic
    "parse_unparse.clj"
    157]
   [typed.clj.checker.parse_unparse$parse_clj
    invoke
    "parse_unparse.clj"
    156]
   [clojure.lang.Var invoke "Var.java" 384]
   [clojure.core.typed$add_tc_type_name$fn__2039$fn__2042$fn__2043$fn__2044$fn__2045
    invoke
    "typed.clj"
    394]
   [typed.clj.checker.parse_unparse$with_parse_ns_STAR_
    invokeStatic
    "parse_unparse.clj"
    109]
   [typed.clj.checker.parse_unparse$with_parse_ns_STAR_
    invoke
    "parse_unparse.clj"
    106]
   [clojure.lang.Var invoke "Var.java" 388]
   [clojure.core.typed$add_tc_type_name$fn__2039$fn__2042$fn__2043$fn__2044
    invoke
    "typed.clj"
    394]
   [clojure.lang.AFn applyToHelper "AFn.java" 152]
   [clojure.lang.AFn applyTo "AFn.java" 144]
   [clojure.core$apply invokeStatic "core.clj" 667]
   [clojure.core$with_bindings_STAR_ invokeStatic "core.clj" 1990]
   [clojure.core$with_bindings_STAR_ doInvoke "core.clj" 1990]
   [clojure.lang.RestFn invoke "RestFn.java" 425]
   [clojure.lang.AFn applyToHelper "AFn.java" 156]
   [clojure.lang.RestFn applyTo "RestFn.java" 132]
   [clojure.core$apply invokeStatic "core.clj" 671]
   [clojure.core$bound_fn_STAR_$fn__5818 doInvoke "core.clj" 2020]
   [clojure.lang.RestFn invoke "RestFn.java" 397]
   [clojure.lang.AFn applyToHelper "AFn.java" 152]
   [clojure.lang.RestFn applyTo "RestFn.java" 132]
   [clojure.core$apply invokeStatic "core.clj" 667]
   [clojure.core$with_bindings_STAR_ invokeStatic "core.clj" 1990]
   [clojure.core$with_bindings_STAR_ doInvoke "core.clj" 1990]
   [clojure.lang.RestFn invoke "RestFn.java" 425]
   [clojure.lang.AFn applyToHelper "AFn.java" 156]
   [clojure.lang.RestFn applyTo "RestFn.java" 132]
   [clojure.core$apply invokeStatic "core.clj" 671]
   [clojure.core$bound_fn_STAR_$fn__5818 doInvoke "core.clj" 2020]
   [clojure.lang.RestFn invoke "RestFn.java" 397]
   [typed.cljc.runtime.env_utils$delay_type_STAR_$__GT_f_delay__2667$fn__2668
    invoke
    "env_utils.clj"
    44]
   [clojure.lang.Delay deref "Delay.java" 42]
   [clojure.lang.Delay force "Delay.java" 28]
   [clojure.core$force invokeStatic "core.clj" 767]
   [clojure.core$force invoke "core.clj" 763]
   [typed.cljc.runtime.env_utils$delay_type_STAR_$fn__2671
    invoke
    "env_utils.clj"
    55]
   [typed.cljc.runtime.env_utils$force_type
    invokeStatic
    "env_utils.clj"
    69]
   [typed.cljc.runtime.env_utils$force_type invoke "env_utils.clj" 68]
   [clojure.lang.Var invoke "Var.java" 384]
   [clojure.core.typed$add_tc_type_name$fn__2039$fn__2042$fn__2043
    invoke
    "typed.clj"
    393]
   [clojure.core.typed$add_tc_type_name$fn__2039$fn__2040
    invoke
    "typed.clj"
    386]
   [clojure.lang.AFn applyToHelper "AFn.java" 154]
   [clojure.lang.AFn applyTo "AFn.java" 144]
   [clojure.core$apply invokeStatic "core.clj" 667]
   [clojure.core$with_bindings_STAR_ invokeStatic "core.clj" 1990]
   [clojure.core$with_bindings_STAR_ doInvoke "core.clj" 1990]
   [clojure.lang.RestFn applyTo "RestFn.java" 142]
   [clojure.core$apply invokeStatic "core.clj" 671]
   [clojure.core$bound_fn_STAR_$fn__5818 doInvoke "core.clj" 2020]
   [clojure.lang.RestFn invoke "RestFn.java" 408]
   [clojure.core.typed$add_tc_type_name$fn__2039$fn__2042
    invoke
    "typed.clj"
    391]
   [clojure.lang.AFn applyToHelper "AFn.java" 152]
   [clojure.lang.AFn applyTo "AFn.java" 144]
   [clojure.core$apply invokeStatic "core.clj" 667]
   [clojure.core$with_bindings_STAR_ invokeStatic "core.clj" 1990]
   [clojure.core$with_bindings_STAR_ doInvoke "core.clj" 1990]
   [clojure.lang.RestFn invoke "RestFn.java" 425]
   [clojure.lang.AFn applyToHelper "AFn.java" 156]
   [clojure.lang.RestFn applyTo "RestFn.java" 132]
   [clojure.core$apply invokeStatic "core.clj" 671]
   [clojure.core$bound_fn_STAR_$fn__5818 doInvoke "core.clj" 2020]
   [clojure.lang.RestFn invoke "RestFn.java" 397]
   [typed.cljc.runtime.env_utils$delay_type_STAR_$__GT_f_delay__2667$fn__2668
    invoke
    "env_utils.clj"
    44]
   [clojure.lang.Delay deref "Delay.java" 42]
   [clojure.lang.Delay force "Delay.java" 28]
   [clojure.core$force invokeStatic "core.clj" 767]
   [clojure.core$force invoke "core.clj" 763]
   [typed.cljc.runtime.env_utils$delay_type_STAR_$fn__2671
    invoke
    "env_utils.clj"
    55]
   [typed.cljc.runtime.env_utils$force_type
    invokeStatic
    "env_utils.clj"
    69]
   [typed.cljc.runtime.env_utils$force_type invoke "env_utils.clj" 68]
   [clojure.core.typed.all_envs$name_env$iter__25245__25249$fn__25250
    invoke
    "all_envs.cljc"
    24]
   [clojure.lang.LazySeq sval "LazySeq.java" 42]
   [clojure.lang.LazySeq seq "LazySeq.java" 51]
   [clojure.lang.Cons next "Cons.java" 39]
   [clojure.lang.RT next "RT.java" 713]
   [clojure.core$next__5451 invokeStatic "core.clj" 64]
   [clojure.core.protocols$fn__8249 invokeStatic "protocols.clj" 169]
   [clojure.core.protocols$fn__8249 invoke "protocols.clj" 124]
   [clojure.core.protocols$fn__8204$G__8199__8213
    invoke
    "protocols.clj"
    19]
   [clojure.core.protocols$seq_reduce invokeStatic "protocols.clj" 31]
   [clojure.core.protocols$fn__8236 invokeStatic "protocols.clj" 75]
   [clojure.core.protocols$fn__8236 invoke "protocols.clj" 75]
   [clojure.core.protocols$fn__8178$G__8173__8191
    invoke
    "protocols.clj"
    13]
   [clojure.core$reduce invokeStatic "core.clj" 6886]
   [clojure.core$into invokeStatic "core.clj" 6958]
   [clojure.core$into invoke "core.clj" 6950]
   [clojure.core.typed.all_envs$name_env
    invokeStatic
    "all_envs.cljc"
    21]
   [clojure.core.typed.all_envs$name_env invoke "all_envs.cljc" 19]
   [clojure.core.typed.all_envs$all_envs_clj$fn__25287
    invoke
    "all_envs.cljc"
    36]
   [clojure.lang.AFn applyToHelper "AFn.java" 152]
   [clojure.lang.AFn applyTo "AFn.java" 144]
   [clojure.core$apply invokeStatic "core.clj" 667]
   [clojure.core$with_bindings_STAR_ invokeStatic "core.clj" 1990]
   [clojure.core$with_bindings_STAR_ doInvoke "core.clj" 1990]
   [clojure.lang.RestFn invoke "RestFn.java" 425]
   [clojure.core.typed.all_envs$all_envs_clj
    invokeStatic
    "all_envs.cljc"
    34]
   [clojure.core.typed.all_envs$all_envs_clj invoke "all_envs.cljc" 32]
   [clojure.lang.Var invoke "Var.java" 380]
   [clojure.core.typed$envs invokeStatic "typed.clj" 1411]
   [clojure.core.typed$envs invoke "typed.clj" 1401]
   [example.core$_main invokeStatic "core.clj" 8]
   [example.core$_main doInvoke "core.clj" 5]
   [clojure.lang.RestFn invoke "RestFn.java" 397]
   [clojure.lang.Var invoke "Var.java" 380]
   [user$eval140 invokeStatic "form-init10603327060393245003.clj" 1]
   [user$eval140 invoke "form-init10603327060393245003.clj" 1]
   [clojure.lang.Compiler eval "Compiler.java" 7194]
   [clojure.lang.Compiler eval "Compiler.java" 7184]
   [clojure.lang.Compiler load "Compiler.java" 7653]
   [clojure.lang.Compiler loadFile "Compiler.java" 7591]
   [clojure.main$load_script invokeStatic "main.clj" 475]
   [clojure.main$init_opt invokeStatic "main.clj" 477]
   [clojure.main$init_opt invoke "main.clj" 477]
   [clojure.main$initialize invokeStatic "main.clj" 508]
   [clojure.main$null_opt invokeStatic "main.clj" 542]
   [clojure.main$null_opt invoke "main.clj" 539]
   [clojure.main$main invokeStatic "main.clj" 664]
   [clojure.main$main doInvoke "main.clj" 616]
   [clojure.lang.RestFn applyTo "RestFn.java" 137]
   [clojure.lang.Var applyTo "Var.java" 705]
   [clojure.main main "main.java" 40]],
  :cause
  "Internal Error (typed/cljc/runtime/env_utils.clj:76:3) Cannot resolve type: s/Str\nHint: Is s/Str in scope in namespace `typed.cljc.runtime.env-utils`?",
  :data
  {:type-error :clojure.core.typed.errors/internal-error,
   :env
   {:line 76,
    :column 3,
    :file "typed/cljc/runtime/env_utils.clj",
    :ns typed.cljc.runtime.env-utils}},
  :phase :execution}}
```

**Comment by frenchy64** [comment]
Thanks for the report! Please use the latest snapshot.

**Comment by frenchy64** [comment]
Fixed in 1.1.5.

**Comment by owenRiddy** [comment]
Thankyou
